src/Shared/HelperFunctions.js

```diff
@@ -14,4 +14,12 @@ const calcAvgRating = (ratings) => {
   return avgRating;
 };
 
-export default calcAvgRating;
+export const defaultStyleFinder = (stylesArr) => {
+  for (let i = 0; i < stylesArr.length; i++) {
+    if (stylesArr[i]['default?'] === 1) {
+      return stylesArr[i];
+    } else {
+      return stylesArr[0];
+    }
+  }
+};
```

Comment:
This helper function iterates through the styles.result array received from the API and determines which style is the default, if no default define it sets the first style as the default.